sim/midas/src/main/scala/midas/passes/AutoILATransform.scala

```diff
@@ -139,7 +139,7 @@ object AutoILATransform extends Transform with DependencyAPIMigration {
     }
 
     val newTop             = possibleTops.head.asInstanceOf[Module]
-    val newPorts           = newTop.ports.toSet -- oldTop.ports.toSet
+    val newPorts           = newTop.ports.filter(!oldTop.ports.contains(_))
```

Comment:
Isn't the `--` still valid? Why do it this way?

Reply:
`--` is not a member of `Seq`

Reply:
Can't you convert things to a set and then use the same operator?

Reply:
I'm trying to avoid sets altogether, since traversal order can break tests. 

Since this is the only place where the set conversion happens, it's cleaner to just to the filter.